src/glyphs.js

```diff
@@ -91,7 +95,8 @@ const processMessageRequest = req => {
 const updateMessages = (id, path, req) => {
   return db.collection.updateOne(
     {
-      _id: new ObjectId(id)
+      _id: new ObjectId(id),
+      [path]: { $exists: true }
```

Comment:
See how easy it was to be certain you updated this everywhere just by changing it in one function! :)